src/Core/Grid/Query/DoctrineQueryParser.php

```diff
@@ -91,6 +91,16 @@ private function parseStringParameter($value)
         return "'" . addslashes($value) . "'";
     }
 
+    /**
+     * @param int|float $value
+     *
+     * @return string
+     */
+    private function parseNumericParameter($value)
+    {
+        return "'" . $value . "'";
```

Comment:
no need to add `'` when we add a is_numeric to call this function. And of course it will be a numeric in sql query instead of string

Reply:
i had this intention not to add `'`, but i thought maybe its safer to keep it. anyways, ill update it. :+1: